### PR TITLE
template for account consumption and tests

### PR DESCRIPTION
**Pull request by alichaddad**
#### What this PR resolves:
template for getting account consumption

**Fixes**: #https://github.com/Jumpscale/ays9/issues/72

### CODE REVIEW

File: tests/bp_test_templates/basic/account_consumption.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+g8client__env:
+  url: '{environment}'
+  login: '{username}'
+  password: '{password}'
```
Comment (ahussein) [inline tests/bp_test_templates/basic/account_consumption.yaml]: This is not how the test runner will replace the placeholders in the test, you need to use <URL>, <LOGIN>, <PASSWORD>, <ACCOUNT>, and <LOCATION> as placeholder in order for the test runner to replace them while running the tests.

Comment (alichaddad) [reply tests/bp_test_templates/basic/account_consumption.yaml]: Is there examples for how it should be done?

Comment (ahussein) [reply tests/bp_test_templates/basic/account_consumption.yaml]: I did not have the time to add documentation yet, but you can check https://github.com/Jumpscale/ays9/blob/master/tests/bp_test_templates/basic/vdc_ovc/001_test_create_vdc.yaml for an example

File: templates/ovc/account/README.md [hunk 2]
```diff
@@ -22,6 +22,16 @@ This actor template is responsible for creating an account on any openVCloud env
...
 - maxDiskCapacity: The limit on the disk capacity that can be used by the account. Default: -1 (unlimited)
 
+- consumptionFrom: determines the start date of the required period to fetch the account consumption info from. If left empty will be creation time of the account.
+
+- consumptionTo: determines the end date of the required period to fetch the account consumption info from. If left empty will be consumptionfrom + 1 hour.
+
+- consumptionData: consumption data will be saved here as series of bytes which represents a zip file. example of writing the data:
+```
+with open('/tmp/account.zip', 'wb') as f:
+    f.write(service.model.data.consumptionData)
```
Comment (rkhamis) [inline templates/ovc/account/README.md]: please update the documentation to reflect usage of the API endpoint =/

**Pull request merged**